src/HasMedia/HasMediaThumbsTrait.php

```diff
@@ -24,7 +23,7 @@ public function getThumbs200ForCollection( string $mediaCollectionName ) {
 					return $conversion->getName() == 'thumb_200';
 				} )->count() > 0;
 		} )->map( function ( $medium ) use ( $mediaCollection ) {
-			return [
+            return [
```

Comment:
Also here the formatting does not looks ok.

Reply:
Same for this line.